lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
I talked this over with @jpinsonault when he implemented it. Basically, this class uses a List as its back-end. The Java re-implements all of these methods for SimpleConfigList, and the re-implemented versions just call through to the corresponding functions on `value`. This forwardable stuff basically forwards those method calls directly to @value so that we don't have to do something like 

```
def include? (param)
    @value.include? (param)
end
```